zone/bot.cpp

```diff
@@ -9830,4 +9812,30 @@ void Bot::SendSpellAnim(uint16 target_id, uint16 spell_id)
 	entity_list.QueueCloseClients(this, &app, false, RuleI(Range, SpellParticles));
 }
 
+float Bot::GetBotCasterMaxRange(float melee_distance_max) {// Calculate caster distances
+	float caster_distance_max = 0.0f;
+	float caster_distance_min = 0.0f;
+	float caster_distance = 0.0f;
+	{
+		if (!GetBotCasterRange()) {
+			if (GetLevel() >= GetStopMeleeLevel() && GetClass() >= WARRIOR && GetClass() <= BERSERKER) {
+				caster_distance_max = MAX_CASTER_DISTANCE[GetClass() - 1];
+			}
+		}
+		else {
+			caster_distance_max = GetBotCasterRange() * GetBotCasterRange();
```

Comment:
The else can be removed and go before the if check

```cpp
caster_distance_max = GetBotCasterRange() * GetBotCasterRange();
if (!GetBotCasterRange()) {
	if (GetLevel() >= GetStopMeleeLevel() && GetClass() >= WARRIOR && GetClass() <= BERSERKER) {
		caster_distance_max = MAX_CASTER_DISTANCE[GetClass() - 1];
	}
}
```

Reply:
Cleaned up.